segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/HDFSIntegrationTest.java

```diff
@@ -22,10 +22,12 @@
 import org.apache.hadoop.hdfs.MiniDFSCluster;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
 
 /**
  * End-to-end tests for SegmentStore, with integrated Storage and DurableDataLog.
  */
+@Ignore
```

Comment:
Please add some background information to make it easier to bring this test back in the future, if at all.

Reply:
The PR and commit message already provides background.

Reply:
Then why not just add the explanation below into comment? 
In the future when people get back to this test for whatever reason, they will wonder what to do with it - bring it back? keep ignoring? improve it? or simply delete it?
PR and commit message don't stay with code always.

Reply:
Kevin I've already fixed it as per your suggestion!